frontend/test/unit/specs/components/v-content-report-form.spec.js

```diff
@@ -84,42 +83,48 @@ describe("VContentReportForm", () => {
   })
 
   it("should render thank you note when report is sent", async () => {
-    const { getByText } = render(VContentReportForm, options)
+    const { queryByText } = render(VContentReportForm, options)
```

Comment:
In some lines here eslint shows warnings for places where you need to add `expect(...).toBeInTheDocument()`

Reply:
Yes! @ZeroPlayerRodent here's the latest errors on this branch:

https://github.com/WordPress/openverse/actions/runs/5380488628/jobs/9763271109

```shell
/home/runner/work/openverse/openverse/frontend/test/unit/specs/components/v-content-report-form.spec.js
Warning:   75:3  warning  Test has no assertions  jest/expect-expect
```